doc/source/faq.rst

```diff
@@ -71,7 +71,7 @@ See :ref:`software_using_gdal`:
 What compiler can I use to build GDAL/OGR?
 ++++++++++++++++++++++++++++++++++++++++++++++++
 
-GDAL/OGR can be compiled with a C++11 capable compiler.
+GDAL/OGR must be compiled with a C++17 capable compiler since GDAL 3.9 (C++11 in previous versions)
```

Comment:
For some reason I'm not seeing the "Suggest changes" button, but I would suggest

```
Build requirements are described in :ref:`build_requirements`.
```

Reply:
Good suggestion. Done

> For some reason I'm not seeing the "Suggest changes" button, but I would suggest

Perhaps because you tried from a commit and not for the full diff of the full request ?